borrowing/models.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+from django.db import models
+
+from book.models import Book
+
+
+class Borrowing(models.Model):
+    borrow_date = models.DateField(auto_now_add=True)
+    expected_date = models.DateField()
+    actual_return_date = models.DateField(blank=True, null=True)
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
+
```

Comment:
I think you should add constraints for borrow_date, expected_return_date, and actual_return_date as it was in task